src/components/Compare/CompareTable.tsx

```diff
@@ -211,6 +212,56 @@ const CompareTable = (props: {
     trackCompareEvent(EventAction.OPEN_MODAL, 'Show All Locations');
   };
 
+  /**
+   * Show limited locations (100 max) in the following scenarios:
+   * - "Counties" tab selected on homepage.
+   * - "Metro areas" tab selected on homepage.
+   * - "USA" tab selected on location page.
+   */
+  const showLimitedLocations =
+    (props.isHomepage && props.homepageScope === HomepageLocationScope.STATE) ||
+    (!props.isHomepage && props.geoScope !== GeoScopeFilter.COUNTRY)
+      ? true
+      : false;
```

Comment:
we can nix the ternary and just keep the condition. it will resolve to true or false on its own, so just -
```
const showLimitedLocations =
    (props.isHomepage && props.homepageScope === HomepageLocationScope.STATE) ||
    (!props.isHomepage && props.geoScope !== GeoScopeFilter.COUNTRY)
```